services/hooks/test/api_test.js

```diff
@@ -619,7 +619,7 @@ helper.secrets.mockSuite('api_test.js', ['taskcluster'], function(mock, skipping
       assume(helper.creator.fireCalls).deep.equals([{
         hookGroupId: 'foo',
         hookId: 'bar',
-        context: {firedBy: 'triggerHookWithToken', payload},
+        context: {firedBy: 'triggerHookWithToken', clientId: null, payload},
```

Comment:
can be omitted here and in the previous test, too.

Reply:
it should be then replaced with clientId: undefined in the test for the tests to pass